togetherly/lib/models/person.dart

```diff
@@ -16,11 +16,13 @@ enum ProfileIcon {
 abstract class Person {
   final int? id;
   final int familyId;
+  final int pin;
   final String name;
   final ProfileIcon icon;
 
   const Person({
     this.id,
+    required this.pin,
```

Comment:
This is nit-picky, but for consistency's sake, it might be good to put this parameter below `familyId` like you have it everywhere else.

Reply:
Done!